src/elements/molecules/Accordion/Accordion.jsx

```diff
@@ -81,7 +81,9 @@ export class Accordion__section extends React.Component {
     /** Level attribute overloads the heading tag size of a accordion section (h1-h6) */
     level: PropTypes.string,
     /** align attribute passes its value down to the expand molecule to align expanded content from top or bottom */
-    align: PropTypes.oneOf(['top', 'bottom'])
+    align: PropTypes.oneOf(['top', 'bottom']),
+    /** ID required for corresponding aria-labelledBy */
+    id: PropTypes.string
```

Comment:
Move this prop up the <Accordion> tag, pass it down to the render like normal, but instead of outputting it, lets pass it down to children (aka <Accordion__section>) for developers:

```javascript
  render = () => {
    const {
      tagName: Tag,
      className,
      variant,
      multi,
      id,
      ...attrs
    } = this.props;

    let { children } = this.props;
    children = Object.keys(children).map((i) => React.cloneElement(children[i], { id: `${id}-${i}`, key: i }
```

Move children to be a `let` so we can re-assign and loop over then. Using React.cloneElement we can pass down a unique id off the top level id. This leave us only one spot now to have to fill out an id for accordions, but pass then down to sections as unique ids to meet expands requirements.